app/ducks/walletReducer.js

```diff
@@ -56,7 +56,7 @@ export default function walletReducer(state = getInitialState(), {type, payload}
           ...state.balance
         },
         isLocked: true,
-        transactions: []
+        transactions: new Map()
```

Comment:
Curious: what is the advantage of using a `Map` in this context as opposed to a simple `{}`?

Reply:
The PropTypes checker was throwing an error before the first update.